linux/inventory.yml

```
@@ -17,6 +17,9 @@ staging:
         com.mbta.ctd.primary-instance:
     HSCTDLNXSTG[02:99]:
     PPCTDLNXSTG[01:99]:
+paess-testing:
+  hosts:
+    PPCTDSCUTST01[01:99]:
```

```suggestion
    PPCTDSCUTST[01:99]:
```
or
```suggestion
    PPCTDSCUTST01:
```

You don't need both the hard-coded number and the range.

---

Got it--I went with `PPCTDSCUTST[01:99]`